src/openforms/dmn/api/urls.py

```diff
@@ -24,4 +26,14 @@
         DecisionDefinitionXMLView.as_view(),
         name="dmn-definition-xml",
     ),
+    path(
+        "decision-definitions/inputs",
+        DecisionDefinitionInputView.as_view(),
+        name="dmn-definition-inputs",
+    ),
+    path(
+        "decision-definitions/outputs",
+        DecisionDefinitionOutputView.as_view(),
+        name="dmn-definition-outputs",
+    ),
```

Comment:
does it make sense to make these two separate endpoints? that implies parsing the XML twice, where-as an endpoint to get both the inputs and outputs in a single response sounds more practical to me (also from a UI perspective -> you only have to make the single API call)

Reply:
I went for 2 endpoints because I thought that I would end up doing 2 calls in the frontend anyway (since with Formik the 2 variables tables will end up being 2 'inter related' fields with the plugin/decision id/decision version). But maybe I can try to figure out another way of dealing with the frontend.